benchmarks/animation-bp/app.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+angular.
+  module('animationBenchmark', ['ngAnimate'], config).
```

Comment:
> Come on! I speeded up your animations 5%-7%. And all you care about is where I put a tiny little dot?

No, I like this PR a lot! I'm just providing feedback for some small issues. :)

Good job overall, though, perhaps I should have spelled that out. 👏